mitm_proxy_helpers/proxy.py

```diff
@@ -5,6 +5,7 @@
 import time
 import select
 from distutils import dir_util
+
```

Comment:
Any requirement for the extra space?

Reply:
it was just to separate: python builtins, 3rd party packages, package modules

Reply:
Copy that